TinyLoRa.cpp

```diff
@@ -500,9 +500,11 @@ uint8_t TinyLoRa::RFM_Read(uint8_t RFM_Address) {
               Frame counter for transfer frames.
     @param    Data_Length
               Length of data to be sent.
+    @param    Frame_Port
+              Number of frame port
```

Comment:
Let's make this more verbose:

Port we send data? [TTN supports 0-255, 1 is default]

```
@param Frame_Port
                Frame port to send data from, from 0 to 225.
```